### ISSUE TITLE
Option to open folders in paths as new windows

### ISSUE THREAD

**Issue by Anthropic** [alefragnani/vscode-project-manager#121]
As #46 is based on supporting Microsoft/vscode#396

I would like to suggest the enhancement of adding an option to open folders listed in paths as new windows. This would be very useful for anyone who like the Microsoft way of working even after #46 is complete but adds a very useful solution to allow working with vscode palatable in the meantime.

Thanks for your efforts, great extension :)

**Comment by rbenson** [comment]
On a similar topic, there used to be a setting to automatically open projects in new windows, and this is now gone.

**Comment by alefragnani** [comment]
@rbenson the setting was removed because a new command `List Projects to Open in New Window` has been added sometime ago, and for new users, it appeared to be a _duplicate command_. 

I suggest you to use the `List Projects to Open in New Window` with a _keybinding_ defined, something like `Ctrl + Shift + Alt + P`, and so, you could decide which way to open the project.

In the future, you could simply use `Ctrl + Click` in the original command, and open in new windows, just like `Open Recent` command already exists. But this will have to way VSCode API to evolve, and is already tracked in #76 .

Hope this helps.

**Comment by rbenson** [comment]
Oh I see. I preferred to set it and set forget it approach. Most of the time I want it in a new window. I removed the default binding, and moved it to the new window command.

**Comment by alefragnani** [comment]
I'm closing this feature request because it didn’t receive enough upvotes from the community. Still a big Thank You to you for taking the time to create this issue!

Thanks for your understanding
